UseCases/Groups.md

```diff
@@ -0,0 +1,24 @@
+## Groups
+
+Groups define the basic structure.
+They can have members in the form of `users` and contain `projects` and `groups`(subGroups)
+
+### Group Operations
+
+#### Add Group
+#### Add SubGroups
+#### Get Group
+#### List Groups
+#### Update Group
+#### Remove Group
+#### Remove Subgroup
+#### Deactivate Group
+#### Reactivate Group
+#### Add Project
+#### Remove Project
+
+### Group Member Operations
+
+#### List Group Members
+#### Add User to Group
+#### Remove User from Group
```

Comment:
Some more lists
List Subgroups of Group
List Projects of Group

Reply:
yes you are right